src/containers/permissions/assign-permissions-to-data-set/AssignPermissionsToDataSetModalBody.js

```diff
@@ -0,0 +1,155 @@
+    <StepsController>
+      {
+        ({ step, stepBack, stepNext }) => (
+          <>
+            {
+              step === 0 && (
+                <>
+                  <ModalBody>
+                    <StepSelectDataSet
+                        organizationId={organizationId}
+                        setTargetDataSetId={setTargetDataSetId}
+                        setTargetDataSetTitle={setTargetDataSetTitle}
+                        targetDataSetId={targetDataSetId} />
+                  </ModalBody>
+                  <ModalFooter
+                      onClickPrimary={stepNext}
+                      onClickSecondary={stepBack}
+                      shouldStretchButtons
+                      textPrimary="Continue"
+                      textSecondary="" />
+                </>
+              )
+            }
+            {
+              step === 1 && (
+                <>
+                  <ModalBody>
+                    <StepSelectPermissions
+                        setTargetPermissionOptions={setTargetPermissionOptions}
+                        targetDataSetTitle={targetDataSetTitle}
+                        targetPermissionOptions={targetPermissionOptions} />
+                  </ModalBody>
+                  <ModalFooter
+                      onClickPrimary={stepNext}
+                      onClickSecondary={stepBack}
+                      shouldStretchButtons
+                      textPrimary="Continue"
+                      textSecondary="Back" />
+                </>
+              )
+            }
+            {
+              step === 2 && (
+                <>
+                  <ModalBody>
+                    <StepSelectProperties
+                        assignPermissionsToAllProperties={assignPermissionsToAllProperties}
+                        setAssignPermissionsToAllProperties={setAssignPermissionsToAllProperties}
+                        targetDataSetTitle={targetDataSetTitle}
+                        targetPermissionOptions={targetPermissionOptions} />
+                  </ModalBody>
+                  <ModalFooter
+                      onClickPrimary={stepNext}
+                      onClickSecondary={stepBack}
+                      shouldStretchButtons
+                      textPrimary="Continue"
+                      textSecondary="Back" />
+                </>
+              )
+            }
+            {
+              step === 3 && (
+                <>
+                  <ModalBody>
+                    <StepConfirm
+                        assignPermissionsToAllProperties={assignPermissionsToAllProperties}
+                        assignPermissionsToDataSetRS={assignPermissionsToDataSetRS}
+                        targetDataSetTitle={targetDataSetTitle}
+                        targetPermissionOptions={targetPermissionOptions} />
+                  </ModalBody>
+                  <ModalFooter
+                      isPendingPrimary={assignPermissionsToDataSetRS === RequestStates.PENDING}
```

Comment:
is there a reason we're not using `isPending` from `ReduxUtils` in `lattice-utils`?